_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+# Whitaker Lab Meeting | 1 August 2019
+
+This week’s lab meet was petite but by no means picayune! 
+We discussed ways to learn physics, Neurohackademy, and how to build trust and confidence in a group. 
+Thanks Kirstie, Isla, and Sarah for your tips, suggestions, and a super interesting conversation!
+
+### Celebrations and cool things to share:
+
+Kirstie: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
```

Comment:
I think it looks nice to format the names at the start of these bullet points in bold.

Sorry for being so picky. If you can change this for the others that would be great!

```suggestion
**Kirstie**: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
```